src/terraform/internal-dss/create_grafana_user.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Create a PostgreSQL data source
+curl "http://admin:$ADMIN_PASSWORD@$GRAFANA_HOST:$GRAFANA_PORT/api/datasources" \
+    -H 'Accept: application/json' \
+    -H 'Content-Type: application/json' \
+    --data "{\"name\":\"DSS Database\",\"type\":\"postgres\",\"url\":\"$DATABASE_HOST:$DATABASE_PORT\",\"access\":\"proxy\",\"user\":\"$DATABASE_USER\",\"password\":\"$DATABASE_PASSWORD\",\"jsonData\": {\"database\":\"$DATABASE_DB\",\"sslmode\":\"disable\",\"postgresVersion\":1500}}"
+
+# Create a public user
+curl "http://admin:$ADMIN_PASSWORD@$GRAFANA_HOST:$GRAFANA_PORT/api/admin/users" \
+    -X POST \
+    -H 'Accept: application/json' \
+    -H 'Content-Type: application/json' \
+    --data "{\"name\":\"public\",\"email\":\"$PUBLIC_USER_EMAIL\",\"login\":\"public\",\"password\":\"$PUBLIC_USER_PASSWORD\",\"OrgId\":1}"
```

Comment:
How about changing this to use `public` as the username instead of having a public user email? That will make it easier to log in using `public` username

Reply:
Note org ID isn't necessary:

https://github.com/econia-labs/econia/blob/af85fe5131ceea4ac879637c157f682621eabd66/src/terraform/dss-ci-cd/dss/modules/grafana/main.tf#L114-L130